build/civicrm/setup-plugins/DockerMonolog.civi-setup.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * @file
+ *
+ * Install and configure Monolog to direct all logs to stdout
+ */
+
+if (!defined('CIVI_SETUP')) {
+  exit("Installation plugins must only be loaded by the installer.\n");
+}
+
+\Civi\Setup::dispatcher()
+  ->addListener('civi.setup.init', function (\Civi\Setup\Event\InitEvent $e) {
+    $e->getModel()->extensions[] = 'monolog';
+  });
+
+\Civi\Setup::dispatcher()
+  ->addListener('civi.setup.installDatabase', function (\Civi\Setup\Event\InstallDatabaseEvent $e) {
+    if (!in_array('monolog', $e->getModel()->extensions)) {
+      // in case installing Monolog extension was overridden by user
+      return;
+    }
+
+    \Civi\Setup::log()->info(sprintf('[%s] Handle %s', basename(__FILE__), 'installDatabase'));
+
+    // disable monolog configs packaged in extension
+    \Civi\Api4\Monolog::update(FALSE)
+      ->addWhere('is_active', '=', TRUE)
+      ->addValue('is_active', FALSE)
+      ->execute();
+
+    // create new config to pass all to stdout
+    \Civi\Api4\Monolog::create(FALSE)
+      ->addValue('name', 'docker_std_out')
```

Comment:
I'm Bikeshedding here, but the name is stdout not std_out.

Reply:
`std_out` is used throughout the existing monolog extension code... so I was following that I guess?
 https://lab.civicrm.org/extensions/monolog/-/blob/master/Civi/MonoLog/MonologManager.php?ref_type=heads#L122